pyaedt/hfss.py

```diff
@@ -1435,16 +1445,16 @@ def create_sbr_antenna(
         target_cs : str, optional
             Target coordinate system. The default is ``None``, in which case
             the active coodiante system is used.
-        model_units : str, optional
+        units : str, optional
             Model units to apply to the object. The default is
             ``None``, in which case the active modeler units are applied.
-        parameters_dict : dict, optional
+        parameters : dict, optional
             Dictionary of parameters. The default is ``None``.
         use_current_source_representation : bool, optional
             Whether to use the current source representation. The default is ``False``.
         is_array : bool, optional
             The default is ``False``.
-        antenna_name : str, optional
+        name : str, optional
             Name of the 3D component. The default is ``None``, in which case the
             name is auto-generated based on the antennas type.
```

Comment:
```suggestion
            name is auto-generated based on the antenna type.
```
You can also use "antenna's" type (possessive form), but my suggestion is more readable.